pkg/auth/authz_handler.go

```diff
@@ -52,9 +50,6 @@ func (a *AuthzHandler) CreateAuthorizer() func(*http.Request) error {
 // Authorizer is used to authorize a request after the Auth function was called using the "Auth*" functions
 // and the principal was stored in the context in the "AuthKey" context value.
 func (a *AuthzHandler) Authorizer(request *http.Request) error {
-	if !a.EnableAuth {
```

Comment:
why was this removed?

Reply:
this was just a workaround to make the subsystem work. now that we have wiremock we can actually test the flows :) so we simply create an empty authorizer in CreateAuthorizer in case EnableAuth is false.

Reply:
I see.
Just wondered because we do [refrain from full initialization](https://github.com/openshift/assisted-service/blob/6b39652ff43206e1e8957742ded7861367182f5b/pkg/auth/auth_handler.go#L49 
) of AuthHandler (aka empty AuthHandler) when `EnableAuth` is set to `False` (also [checked upon incoming requests](https://github.com/openshift/assisted-service/blob/6b39652ff43206e1e8957742ded7861367182f5b/pkg/auth/auth_handler.go#L222)).

Can / should we do the same here?

Reply:
Yeah, we do something similar for authz here: https://github.com/openshift/assisted-service/blob/6b39652ff43206e1e8957742ded7861367182f5b/pkg/auth/authz_handler.go#L43

Reply:
ack, looks good.